src/sockperf.cpp

```diff
@@ -2948,9 +2919,11 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
     int sock_type = SOCK_DGRAM;
     int curr_fd = 0, last_fd = 0;
 #ifdef NEED_REGEX_WORKAROUND
-    regex_t regexpr;
+    regex_t regexpr_ip;
+    regex_t regexpr_unix;
 #else // NEED_REGEX_WORKAROUND
-    const std::regex regexpr(IP_PORT_FORMAT_REG_EXP);
+    const std::regex regexpr_ip(IP_PORT_FORMAT_REG_EXP);
+    const std::regex regexpr_unix(UNIX_DOMAIN_SOCKET_FORMAT_REG_EXP);
```

Comment:
Please add UDS socket example into a comment before this function (as already done for IPv4/IPv6 addresses.